app/Http/Controllers/CapacityController.php

```
@@ -38,14 +38,23 @@ public function __construct(CacheService $cacheService, ResourceService $resourc
 
+    /**
+     * Return the paginated collection with the next twelve months and the regions
+     * associated with the resources
+     *
+     * @param int $page
+     * @param int $perPage
+     * @return \Illuminate\Pagination\LengthAwarePaginator
+     */
     public function index(Request $request)
     {
         $user = auth()->user();
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$user'. (undefined)

(UnusedLocalVariable)